package.json

```diff
@@ -29,22 +29,22 @@
     "@antora/cli": "^2.1.2",
     "@antora/site-generator-default": "^2.1.2",
     "babel-eslint": "10.0.1",
-    "eslint": "^7.9.0",
-    "eslint-config-prettier": "^6.10.1",
+    "eslint": "^7.19.0",
```

Comment:
Could you run `yarn` from the root of the repo and commit any changes in `yarn.lock`?